org.eclipse.m2e.binaryproject/src/org/eclipse/m2e/binaryproject/internal/AbstractBinaryProjectsImportJob.java

```diff
@@ -26,7 +26,7 @@
 public abstract class AbstractBinaryProjectsImportJob extends Job {
 
   public AbstractBinaryProjectsImportJob() {
-    super("Import binary projects");
+    super("Import binary projects"); //$NON-NLS-1$
```

Comment:
This should typically be internationalized; hence `//$NON-NLS-1$` is erroneous here.